### ISSUE TITLE
Close upload loophole for open projects

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#1291]
Requires ISPyB/SSH fix. #1264

**Comment by alanbchristie** [comment]
This issue can be developed but cannot be released until #1264 is resolved

**Comment by alanbchristie** [comment]
When uploading compound sets the user currently is required to provide only the `Target` title. Should they now be required to also provide the `Proposal` (Targets can belong to multiple proposals). It's not completely necessary as I can iterate through all the Proposals a Target belongs to.

**Comment by alanbchristie** [comment]
Code is implemented but cannot be released until #1264 is resolved

**Comment by phraenquex** [comment]
Hi @alanbchristie yes good point, they need both `Target` and `target_access_string` (Visit-ID).

@mwinokan  squawk if I got it worng.

**Comment by mwinokan** [comment]
Yes that seems sensible. Could the proposal be a dropdown in the upload UI, once the target has been specified?
